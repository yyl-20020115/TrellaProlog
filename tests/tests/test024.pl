:-initialization(main).
:-set_prolog_flag(double_quotes,codes).

main :- S="a b c", write(S), nl.
