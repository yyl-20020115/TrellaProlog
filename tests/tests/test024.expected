[97,32,98,32,99]
